projectiles/huge_laser.gd

```diff
@@ -0,0 +1,2 @@
+class_name HugeLaser
+extends ProjectileEffect
```

Comment:
:wine_glass: :cheese: 

This is not actually needed with the current implementation.

It WILL be needed if we want to use the Node class type instead of a PackedScene for effects management. I.e. you can't do `HugeLaser.new()` unless you define the class_name in a script.
But if you load a `PackedScene` from the scene path, this is totally superfluous boilerplate.